src/explore-education-statistics-common/src/modules/find-statistics/components/__tests__/KeyStat.test.tsx

```diff
@@ -107,9 +107,9 @@ describe('KeyStat', () => {
       );
 
       expect(
-        screen.getByRole('button', {
-          name: 'What is the number of applications received?',
-        }),
+        screen.getByTestId(
+          'Expand Details Section What is the number of applications received?',
```

Comment:
I don't think this test should need to change as it's for when there is a dataDefinitionTitle.

Reply:
Looks like this test fails on the following without this change: 

```typescript
  <summary
              aria-controls="details-content-1"
              aria-expanded="false"
              class="govuk-details__summary"
              role="button"
              tabindex="0"
            >
              <span
                class="summaryText govuk-details__summary-text"
                data-testid="Expand Details Section What is the number of applications received?"
                id="details-heading-details-content-1"
              >
                What is the number of applications received?
                <span
                  class="govuk-visually-hidden"
                >

                  for Number of applications received
                </span>
              </span>
            </summary>
```

Reply:
Sorry I wasn't very clear, I meant that it shouldn't need to change if we have it so the extra visually hidden text is only added when there's not a `dataDefinitionTitle`.  
As `dataDefinitionTitle` is controlled by the analysts, and looking at the example you screenshotted above generally makes enough sense by itself, we shouldn't append the extra text to it as it won't necessarily make sense, as in the test here which would read " What is the number of applications received? for Number of applications received".